apps/web/src/components/editor/index.tsx

```diff
@@ -382,7 +382,6 @@ function EditorView({
         }}
         options={{
           readonly: session?.type === "readonly" || session?.type === "deleted",
-          onRequestFocus: () => toggleProperties(false),
```

Comment:
We want to hide properties pane when you focus the editor, however for TOC we want to keep it open.